galley/cmd/galley/cmd/root.go

```diff
@@ -89,6 +89,10 @@ func GetRootCmd(args []string, printf, fatalf shared.FormatFn) *cobra.Command {
 				fatalf("Invalid validationArgs: %v", err)
 			}
 
+			if !validationArgs.EnableValidation {
```

Comment:
@ayj I tested this case. As you know that my changes will remove the `validatingwebhookconfiguration.yaml` in `configmap` w/validation=false so that the old pod w/validation=true cannot reload the validatingwebhookconfiguration.

Reply:
configmap propagation is not instantaneous. The old pod might still retain the view of the previous webhook configuration before it's deleted. I would expect your code to mostly work, but I'm concerned about the rare corner case when things race and we leave the cluster in a broken state.

Reply:
@ayj Add new pod passes the readiness check then delete webhook configuration. please review. Thanks.

Reply:
I believe the cleanup validation code in the new pod may run before the pod is ready. Therefore the old pod still has an opportunity to overwrite the config before it exists.